helix-core/src/main/java/org/apache/helix/messaging/handling/HelixTaskExecutor.java

```diff
@@ -788,8 +793,8 @@ public void onMessage(String instanceName, List<Message> messages,
     List<MessageHandler> nonStateTransitionHandlers = new ArrayList<>();
     List<NotificationContext> nonStateTransitionContexts = new ArrayList<>();
 
-    // message read
-    List<Message> readMsgs = new ArrayList<>();
+    // message to be updated in ZK
+    List<Message> updatingMsgs = new ArrayList<>();
```

Comment:
this is a little confusing. what's wrong with leaving it as `readMsgs`? `updatingMsgs` is a confusing name since messages don't update themselves. 

or `msgsToBeHandled` or `msgsToHandle` could also be as equally descriptive

Reply:
readMsgs is no longer an accurate name with this change.
How about msgsToBeUpdated ?